LayoutTests/inspector/console/js-completions.html

```diff
@@ -52,6 +52,27 @@
         }
     });
 
+    suite.addTestCase({
+        name: "console.jsCompletions.completionOrdering",
+        description: "Test that the JavaScript completions are ordered correctly.",
+        async test() {
+            await InspectorTest.evaluateInPage(`myObject = {
+                1: 42, 2: 42, 10: 42, 20: 42, aa: 42, ab: 42, Aa: 42, Ab: 42, __aa__: 42, __ab__: 42
```

Comment:
NIT: rather than do this as part of the test, id just declare this at the top of the test file
```js
window.myObject = {
    1: 42,
    ...
    __ab__: 42,
};
```

Reply:
the main reason being that then other test cases can use it too if they want

also, having a test case "pollute" the global object can be harder to maintain if subsequent tests also want to do the same (e.g. what if we added another test case that also wanted to create a custom global `myObject`?)

FWIW we do this a lot with tests involving DOM nodes or CSS stuff and almost always prefer just having it exist before the test even runs (i.e. part of the file instead of evaluated after)

unless you're specifically trying to observe the act of "polluting" the global object id suggest avoiding having that as part of the test case :)